bigbluebutton-html5/imports/ui/components/app/component.jsx

```diff
@@ -349,11 +350,11 @@ class App extends Component {
         </section>
         <PollingContainer />
         <ModalContainer />
-        <AudioContainer />
+        {micsLocked ? null : <AudioContainer />}
```

Comment:
Viewers are still able to join listenOnly when mic are locked through clicking the join audio button. Since the viewers only option is to join listenOnly, it is not necessary to display the audio modal.
`return micsLocked ? Service.joinListenOnly() : mountModal(<AudioModalContainer />);`